lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
Looking at these, do we want to validate that the "extra headers" passed in do not contain any of these?